website/siteConfig.js

```diff
@@ -170,6 +172,8 @@ const siteConfig = {
     headerLinks: [
         {href: '/#demo', label: 'Try'},
         {doc: 'node', label: 'Docs'},
+        // always link to the latest version page
+        {href: 'https://katex.org/versions.html', label: 'v' + version},
```

Comment:
Will the old versions on netlify be able to link back to the current version?

Reply:
Only versions after this PR, so it would work on versions after `v0.13.0`.